notifications/src/main/kotlin/com/amazon/opendistroforelasticsearch/notifications/util/XContentHelpers.kt

```diff
@@ -61,3 +62,12 @@ internal fun XContentBuilder.objectIfNotNull(name: String, xContentObject: ToXCo
     }
     return this
 }
+
+internal fun <T : Any> XContentParser.objectList(parse: KFunction1<XContentParser, T>): MutableList<T> {
```

Comment:
instead of using reflection, can we use lambda/block?

Reply:
fixed